### ISSUE TITLE
Consider Staying on Fedora 28

### ISSUE THREAD

**Issue by Tjzabel** [RITlug/TigerOS#150]
# Dilemma
Due to many issues with the F28 --> F29 upgrade process, it may make sense to stay on F28, iron out all our current issues, and *then* move over to F29. This would allow for a more streamlined process during the transition. Currently, there are a few issues that need to be ironed out on the F28 side of things. 

# Pros and Cons of Choices

### Pros
* This would allow us to iron out all issues, and get a beta release out relatively quickly

### Cons
* Would be out of date with current upstream
* Still need to eventually move over to F29

### Pros
* Allow us to stay up to date with upstream

### Cons
* Would take a lot of work, and most likely the entire semester

**Comment by ct-martin** [comment]
Given that F30 will [release late April or early May](https://fedoraproject.org/wiki/Releases/30/Schedule) and at that point F28 will be EOL, I'm not sure how much of a choice we have. This is unfortunately a downside of Fedora's fast release schedule.

It may make sense to split efforts and have package dev continue on F28 but have some people working on F29. We can talk about more in a meeting.

**Comment by Tjzabel** [comment]
Hmm, so at this point, we are going to stick with our Fedora 29 goal. 

Currently, on F28, everything is working except for minor UI issues. The purpose of this issue was to possibly stay on F28 in order to fix all issues *before* going over to F29. However, since just about all issues except for very minor ones are fixed on F28, I think it is time to start thinking about an F29 sprint.

With that being said, I've created a [project board](https://github.com/RITlug/TigerOS/projects/7?fullscreen=true) to keep track of our current sprints. 

Now that we've decided on continuing our F29 goals, and since F30 is indeed still going to be a 6-month release, we are going to close this issue, and focus our efforts on getting F29 stable.
